documentation/DPG-quickstart.md

```diff
@@ -0,0 +1,89 @@
+# Getting Started: Generate JavaScript/TypeScript DPG Libraries with TypeSpec
```

Comment:
I notice the pr is duplicated with mine https://github.com/Azure/azure-sdk-for-js/pull/27674.